### DISCUSSION TITLE
Locations of predicted transcription factor binding sites within regions

### DISCUSSION THREAD

**Discussion by Martijnvanderwerff** [aertslab/scenicplus#411] [Q&A] [answered]
Dear all,

First of all thank you for this great tool! I have a question regarding the predicted transcription factor binding sites. We are interested in a region were we observe two predicted TFBS. Is there an output file were I can find the exact locations of these sites within the region? Or should I look for these locations in a separate downstream analysis? 

Kind regards,
Martijn

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @Martijnvanderwerff 

Unfortunately SCENIC+ does not provide information on the exact location of the binding sites.
It can only tell you which regions (of 500 bp by default) are predicted to be bound by the TF.

Finding the exact binding site location  should this be done in a separate downstream analysis (for example you can use Cluster-Buster for this, to score predicted target regions for motifs of the TF(s) of interest).

I hope this helps?

Best,

Seppe

**Reply by Martijnvanderwerff** [reply]
@SeppeDeWinter Thank you for the clarification! Cluster-Buster was also a solution we were thinking about.
